apis/server/container_bridge.go

```diff
@@ -216,28 +218,48 @@ func (s *Server) attachContainer(ctx context.Context, rw http.ResponseWriter, re
 }
 
 func (s *Server) getContainers(ctx context.Context, rw http.ResponseWriter, req *http.Request) error {
+	option := &mgr.ContainerListOption{
+		All: httputils.BoolValue(req, "all"),
+	}
+
 	metas, err := s.ContainerMgr.List(ctx, func(meta *mgr.ContainerMeta) bool {
 		return true
-	})
+	}, option)
 	if err != nil {
 		return err
 	}
 
 	containerList := make([]types.Container, 0, len(metas))
+
 	for _, m := range metas {
+		t, _ := time.Parse(utils.TimeLayout, m.Created)
 		container := types.Container{
 			ID:         m.ID,
 			Names:      []string{m.Name},
-			Status:     string(m.State.Status),
 			Image:      m.Config.Image,
 			Command:    strings.Join(m.Config.Cmd, " "),
-			Created:    m.State.StartedAt,
+			Created:    t.UnixNano(),
 			Labels:     m.Config.Labels,
 			HostConfig: m.HostConfig,
 		}
+
+		if m.State.Status == types.StatusRunning || m.State.Status == types.StatusPaused {
```

Comment:
I think in the future this part of code could be encapsulated into a single function, like `genStatus(State)`.